website/docs/r/ssmcontacts_contact_channel.html.markdown

```diff
@@ -57,7 +57,7 @@ The following arguments are required:
 
 - `delivery_address` - (Required) Block that contains contact engagement details. See details below.
 
-- `name` - (Required) Name of the contact channel.
+- `name` - (Required) Name of the contact channel. 1-50 characters, alphanumeric, _ (underscore), - (hyphen), and spaces.
```

Comment:
Similar phrasing tweaks to the other two. I also noted in the [API documentation](https://docs.aws.amazon.com/incident-manager/latest/APIReference/API_SSMContacts_CreateContactChannel.html) that periods are accepted, and the maximum length is 255.

```suggestion
- `name` - (Required) Name of the contact channel. Must be between 1 and 255 characters, and may contain alphanumerics, underscores (`_`), hyphens (`-`), periods (`.`), and spaces.
```